src/pages/signin/LoginForm.js

```diff
@@ -102,19 +125,21 @@ class LoginForm extends React.Component {
 
     render() {
         return (
-            <>
+            <View style={!this.props.visible && styles.visuallyHidden}>
                 <View style={[styles.mt3]}>
                     <ExpensiTextInput
+                        ref={el => this.input = el}
                         label={this.props.translate('loginForm.phoneOrEmail')}
                         value={this.state.login}
-                        autoCompleteType="email"
+                        autoCompleteType="username"
```

Comment:
Should we create a constant fo this as well?

Reply:
Does not seem necessary as it should be probably be used in one place. But we can.

Reply:
Good point. it's a NAB for me in that case - I'll leave it up to you to decide.